### PR TITLE
Develop panel to include all results form a given query

### PR DESCRIPTION
**Pull request by casanted**
- Updated displayPanel() function to create and append a child element for each search result obtained from query
- Initially hides panel, only reveals panel once there are search results to be displayed
- Refreshes results list each time, so new results are not just added to old results

image

### CODE REVIEW

File: capstone/src/main/webapp/script.js [hunk 1]
```diff
@@ -142,19 +145,27 @@ function callback(results, status) {
 
 function setMarker(place) {
   const marker = new google.maps.Marker({
-    map: _map,
+    map: map,
```
Comment (terencedignon) [inline capstone/src/main/webapp/script.js:148]: I think this should still be _map, but could be wrong

File: capstone/src/main/webapp/index.html [hunk 2]
```diff
@@ -32,11 +32,12 @@
...
-        <div id="panel">
-        <h2 id="restaurant-info"></h2>
-        </div>
+      <div id="panel">
+        <h2>Results</h2>
+        <h4 id="restaurant-results"></h4>
+      </div>
 
-        <div id="map"></div>
+      <div id="map"></div>
```
Comment (terencedignon) [inline capstone/src/main/webapp/index.html:40]: nice catch

File: capstone/src/main/webapp/script.js [hunk 3]
```diff
@@ -118,6 +118,9 @@ function createMap() {
 }
 
 function getSearchResults() {
+  document.getElementById("map").style.width = "75%";
```
Comment (terencedignon) [inline capstone/src/main/webapp/script.js:121]: OOC why is width and style and whatnot set in the script? I would expect it in the CSS.

Comment (graceehuu) [reply capstone/src/main/webapp/script.js:121]: I think this is changing the map view from 100% -> 75%, so 25% of screen width is used for panel width

File: capstone/src/main/webapp/script.js [hunk 4]
```diff
@@ -142,19 +145,27 @@ function callback(results, status) {
...
+  const locationElement = document.getElementById("restaurant-results");
+  locationElement.appendChild(createLocationElement(name));
+}
+ 
+function createLocationElement(placeName) {
+  const locElement = document.createElement('p');
+  locElement.className = 'place';
+ 
+  const mainElement = document.createElement('span');
+  mainElement.innerText = placeName;
```
Comment (graceehuu) [inline capstone/src/main/webapp/script.js:165]: clean organization!

**Comment by graceehuu** [comment]
LGTM!

File: capstone/src/main/webapp/script.js (hunk 3 above)
Comment (terencedignon) [reply capstone/src/main/webapp/script.js:125]: Thanks Grace, LGTM.

**Pull request merged**